### DISCUSSION TITLE
Infolist with featured image

### DISCUSSION THREAD

**Discussion by mloeff** [filamentphp/filament#11038] [Help]
### Package

Infolist builder

### Package Version

v3.2.0

### How can we help you?

Hi - I'm pretty new to Filament, so maybe I'm missing something.  I have a model called Listing, it has several related models - including ListingMedia.  ListingMedia is essentially a list of images for the Listing - one of those images is marked as the featured image (bool featured_image in the model).  Now I'm trying to display the image using Infolist, but nothing is displayed for the featured image.  Here is the part of code I'm trying to get to work (I'm using Infolist in a livewire (v3) component).  I am currently only pulling the featured image with media, so it's redundant to check for the featured again (but I will have all media in there eventually):

    public function Infolist(Infolist $infoList): Infolist
    {
        $this->listing = Listing::where('id', $this->listing_id)
            ->with([
                'media' => function ($query) {
                    $query->where('is_featured', true)->limit(1);
                }
            ])
            ->first();

        $featuredImage = $this->listing->media()->where('is_featured', true)->first();
        $featuredImageUrl = $featuredImage ? $featuredImage->media_url : null;
        \Log::info('DEBUG: featuredImageUrl=' . $featuredImageUrl);

        return $infoList
            ->record($this->listing)
            ->schema([
                Tabs::make('Listing Details')->tabs([
                    Tabs\Tab::make('Details')->schema([
                        TextEntry::make('title'),
                        TextEntry::make('description'),
                    ]),
                    Tabs\Tab::make('Photos')->schema([
                        ImageEntry::make('featured_image')
                            ->url($featuredImageUrl),
                    ]),
                ])
            ]);
    }

Blade template for the livewire component:
<div class="container mx-auto p-4 my-4 bg-white rounded-md shadow-md">
    {{ $this->infolist }}
</div>

The title and description fields show, but the featured image does not.  I log the featuredImageUrl above and it appears to be correct (http://localhost:8882/storage/listing_attachments/01HMFJQP6H1BZXQVRTVFA9ZSGF.png) - I can go directly to the link in my browser and the image shows just fine, but it's still not rendered in the Infolist.

What am I doing wrong?
Thank you!
-Michael

**Comment by mloeff** [comment]
I listed the wrong bool flag in my explanation- it's is_featured, not featured_image.

**Comment by trinketbox** [comment]
I could be wrong, but the url is used to create a link for the image, not display it. By default it's looking in the database for a field called 'featured_image'. If nothing is found no image is displayed. However, it might still be that the image that doesn't appear has a link to the image you tried to create within the source.

You could try the below instead: 

`
->defaultImageUrl(url($featuredImageUrl))
`
